rosmon_core/src/main.cpp

```diff
@@ -326,10 +323,33 @@ int main(int argc, char** argv)
 
 		// Disable direct logging to stdout
 		ros::console::backend::function_print = nullptr;
+                
+                std::string logFile; 
 
 		// Open logger
-		if(logFile.empty())
+		if(const char* logDir = std::getenv("ROSMON_LOG_PATH"))
```

Comment:
why add env var, instead of param?

Reply:
Didn't try param. @gauthamm should I try param?

Reply:
When you mean param, do you mean rosparam (which is not possible here) or as a configuration param input like --disable-ui --robot etc. If its the case of configuration input, I would prefer that since its cleaner. However Linda said there were some issues with extracting the configuration variable set there in this place. Setting getenv is Ok, but we would need to have a explicit mention in readme and this will not show up as a parameter when we do `mon --help`

Reply:
I will try with configuration param again. The reason why it didn't work might be because I didn't add it to apis.py I only called it at the highest level.